package.json

```diff
@@ -25,8 +32,11 @@
     "css-loader": "^3.5.3",
     "less": "^2.7.1",
     "less-loader": "^6.1.0",
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^3.0.2",
     "style-loader": "^1.2.1",
     "terser-webpack-plugin": "^3.0.2",
+    "ttab": "^0.6.1",
```

Comment:
Is `ttab` being used by the app, or is this just a convenience dependency? I didn't see it being referenced anywhere else.

Reply:
Accidentally left it in when I was trying something out!